### PR TITLE
refactor: Camunda License 1.0

### LINKED ISSUE #18104: Relicense to Camunda License 1.0
**Description**

```[tasklist]
### Tasks
- [x] Remove Zeebe Community License from the repository
- [x] Include new [Camunda License](https://docs.google.com/document/d/1xO8c6LgdKI5Dj3bzzgcL-l0xnQb-bs5mnrJXvuh3tcs/edit)
- [x] Update source code license headers (Zeebe, Tasklist, Operate)
- [x] Update the Readme and the contributor guidelines
```

### PR DESCRIPTION
**Pull request by romansmirnov**
## Description

* Removes Zeebe Community License 1.1 from the repository
* Removes temporary Operate and Tasklist source available license from the repository (deletes the obsolete `license` folder)
* Introduces new [Camunda License 1.0](https://docs.google.com/document/d/1xO8c6LgdKI5Dj3bzzgcL-l0xnQb-bs5mnrJXvuh3tcs/edit)
* Updates all source code license headers (Zeebe, Tasklist, and Operate) accordingly to link the Camunda License 1.0
* Includes Camunda License 1.0 in distributions

## Related issues

closes #18104

### CODE REVIEW

File: licenses/CAMUNDA-LICENSE-1.0.txt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+Definitions
+You refer to the individual or entity agreeing to these terms.
+Use means any action concerning the Software that, without permission, would make Youliable
+for infringement under applicable copyright law. Use within the meaning of this License
+includes, but is not limited to, copying, distribution (with or without modification),
+making available to the public, and modifying the Software.
+
+Non-Production Environment means a setting in which the Software is used for development, stagging,
```
Comment (oleschoenburg) [inline licenses/CAMUNDA-LICENSE-1.0.txt:63]: That's looks like a typo, should be "staging", right?

Comment (romansmirnov) [reply licenses/CAMUNDA-LICENSE-1.0.txt:63]: I just copied & pasted it 😄 But yes, I think so, I will adjust it.

**Comment by romansmirnov** [comment]
Quickly rebasing with `main`

**Comment by romansmirnov** [comment]
@Zelldon, I should have caught all the missing places after rebasing.

File: tasklist/archiver/src/main/java/io/camunda/tasklist/archiver/es/AbstractArchiverJobElasticSearch.java [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
- *
- * SUBJECT AS SET OUT BELOW, THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
- * NOTHING IN THIS AGREEMENT EXCLUDES OR RESTRICTS A PARTY’S LIABILITY FOR (A) DEATH OR PERSONAL INJURY CAUSED BY THAT PARTY’S NEGLIGENCE, (B) FRAUD, OR (C) ANY OTHER LIABILITY TO THE EXTENT THAT IT CANNOT BE LAWFULLY EXCLUDED OR RESTRICTED.
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
```
Comment (Zelldon) [inline tasklist/archiver/src/main/java/io/camunda/tasklist/archiver/es/AbstractArchiverJobElasticSearch.java:6]: In all these files we have this short header, this is expected yes? Also it is always referencing to the NOTICE file but the notice file is not including much information :thinking: This is fine?

**Review by ChrisKujawa — APPROVED** [review]
I took mostly random samples which look fine (I mean mostly the java files are anyway the same and I mostly checked poms and configs :) ) LGTM :rocket:

**Pull request merged**